Formula/goployer.rb

```diff
@@ -0,0 +1,19 @@
+class Goployer < Formula
+  desc "AWS Deployer with Go"
+  homepage "https://goployer.dev"
+  url "https://github.com/DevopsArtFactory/goployer/archive/1.0.1.tar.gz"
+  sha256 "70a37a85a1a18ea10c6522ae9b29d38462625bb14720cd6fd35b8c95c8fdb955"
+  license "Apache-2.0"
+  head "https://github.com/DevopsArtFactory/goployer.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "make"
+    bin.install "out/goployer"
+  end
+
+  test do
+    system "false"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Oh...! I think I need to make version check because goployer is for deployment tool which spend time! Thanks I will fix it

Reply:
You really shouldn't make a version check unless that is all the software does.